scss/components/_progress-bar.scss

```diff
@@ -43,7 +43,9 @@
   .progress {
     @include progress-container;
 
-    @each $name, $color in $foundation-palette {
+    @each $name, $color-name in $foundation-states {
```

Comment:
we need to add `$progess-satets` to make this work like all other components using states.